[2025-10-11T06:30:43.152+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T06:30:43.205+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2000-08-13T00:00:00+00:00 [queued]>
[2025-10-11T06:30:43.220+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2000-08-13T00:00:00+00:00 [queued]>
[2025-10-11T06:30:43.221+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T06:30:43.248+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_create_stg_tables> on 2000-08-13 00:00:00+00:00
[2025-10-11T06:30:43.262+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=13432) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T06:30:43.264+0000] {standard_task_runner.py:64} INFO - Started process 13437 to run task
[2025-10-11T06:30:43.262+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'check_create_stg_tables', 'scheduled__2000-08-13T00:00:00+00:00', '--job-id', '1741', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpbj4z65x2']
[2025-10-11T06:30:43.266+0000] {standard_task_runner.py:91} INFO - Job 1741: Subtask check_create_stg_tables
[2025-10-11T06:30:43.357+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2000-08-13T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T06:30:43.517+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='check_create_stg_tables' AIRFLOW_CTX_EXECUTION_DATE='2000-08-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2000-08-13T00:00:00+00:00'
[2025-10-11T06:30:43.519+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T06:30:43.543+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:30:43.644+0000] {connection.py:43} INFO - Table 'stg_users' in Postgres.
[2025-10-11T06:30:43.646+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:30:43.741+0000] {connection.py:43} INFO - Table 'users' in Postgres.
[2025-10-11T06:30:43.743+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:30:43.842+0000] {connection.py:43} INFO - Table 'stg_movies' in Postgres.
[2025-10-11T06:30:43.844+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:30:43.939+0000] {connection.py:43} INFO - Table 'movies' in Postgres.
[2025-10-11T06:30:43.941+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:30:44.051+0000] {connection.py:43} INFO - Table 'stg_ratings' in Postgres.
[2025-10-11T06:30:44.054+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:30:44.144+0000] {connection.py:43} INFO - Table 'ratings' in Postgres.
[2025-10-11T06:30:44.146+0000] {load.py:32} INFO - Tables exist.
[2025-10-11T06:30:44.147+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T06:30:44.149+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T06:30:44.168+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=check_create_stg_tables, run_id=scheduled__2000-08-13T00:00:00+00:00, execution_date=20000813T000000, start_date=20251011T063043, end_date=20251011T063044
[2025-10-11T06:30:44.244+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T06:30:44.260+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
